pages/index.js

```diff
@@ -5,6 +5,8 @@ import { loadHomePage } from '../store/actions';
 import SignIn from '../components/SignIn';
 import DummyPost from '../components/DummyPost';
 import Post from '../components/Post';
+import PaginationLinks from '../components/PaginationLinks';
+import {useRouter} from "next/router";
```

Comment:
Code style. Also, this line should be moved up to the first group of imports.

Reply:
I suppose I can delete row 9 because it doesn't active now.